examples/demo/src/Login/Login.validation.rules.js

```diff
@@ -0,0 +1,26 @@
+import { LOGIN, PASSWORD, MSG_REQUIRED } from './constants';
+
+const rulesRequired = {
```

Comment:
I think is better to create one object rules that contains differents rules
for example 

const rules = {
  required: {
        message: MSG_REQUIRED
  },
  maxLength: {
    maxLength: max,
    message: 'Le champ contient trop de caractères',
  },
  number: {
    regex: /^[0-9]*$/,
    message: 'Veuillez saisir un nombre',
  }
};

export const loginFormRules = {
  [LOGIN]: [rules.required],
  [PASSWORD]: [rules.required, rules.maxLength(8), rules.number]
};